docs/configuration.md

```diff
@@ -188,39 +184,38 @@ pool_error_threshold: 100
 max_clients_count: 100
 max_clients_deadline: 30s
 
-allowed_access_key_id_prefixes: 
-   - Ck9BHsgKcnwfCTUSFm6pxhoNS4cBqgN2NQ8zVgPjqZDX
-   - 3stjWenX15YwYzczMr88gy3CQr4NYFBQ8P7keGzH5QFn
+allowed_access_key_id_prefixes:
+  - Ck9BHsgKcnwfCTUSFm6pxhoNS4cBqgN2NQ8zVgPjqZDX
+  - 3stjWenX15YwYzczMr88gy3CQr4NYFBQ8P7keGzH5QFn
 ```
 
-| Parameter                        | Type       | SIGHUP reload | Default value  | Description                                                                                                                                                                                                       |
-|----------------------------------|------------|---------------|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `listen_domains`                 | `[]string` |               |                | Domains to be able to use virtual-hosted-style access to bucket.                                                                                                                                                  |
-| `rpc_endpoint`                   | `string`   | yes           |                | The address of the RPC host to which the gateway connects to resolve bucket names (required to use the `nns` resolver).                                                                                           |
-| `resolve_order`                  | `[]string` | yes           | `[dns]`        | Order of bucket name resolvers to use. Available resolvers: `dns`, `nns`.                                                                                                                                         |                                                                                                                                                                           |
```

Comment:
Just removing a string would be easier to handle.

Reply:
Updated